generative_PWM.yml

```diff
@@ -0,0 +1,28 @@
+definitions:
+  datasets:
+    d1:
+      format: Generic
+      params:
+        path: PWM_dataset/sequences.tsv
+        is_repertoire: False
+        paired: False
+        region_type: FULL_SEQUENCE
+  encodings:
+    e1: TextAsInt
+  ml_methods:
+    G1:
+      PWM:
+        cores_for_training: 2
+  reports:
+    GeneratorReport: GeneratorReportPWM
+instructions:
+  machine_learning_instruction:
+    type: GenerativeModel
```

Comment:
To match the previous comment: what does this instruction do with the generative model? Is it training it? Might be nice to have it in the instruction name

Reply:
Could it be something like, GenerativeTrain, and GenerativeLoad? Or should I make it very descriptive, with a long name?